commcare-cloud/commcare_cloud/commcare_cloud.py

```diff
@@ -89,14 +97,30 @@ def has_arg(unknown_args, short_form, long_form):
 
 
 class AnsibleContext(object):
-    def __init__(self):
+    def __init__(self, args):
         self._ansible_vault_password = None
+        self.env = self._build_env(args)
 
     def get_ansible_vault_password(self):
         if self._ansible_vault_password is None:
             self._ansible_vault_password = getpass.getpass("Vault Password: ")
         return self._ansible_vault_password
 
+    def get_environment(self):
```

Comment:
Nit. Took me a while to understand what was happening here because we use the word `environment` in `commcare-cloud` to mean which cluster/instance (e.g. `production`, `enikshay`, etc.). I think if we stick with the name `env_vars` throughout for environment variables, then I think the distinction becomes much clearer.

Reply:
ce662df